rigex-front-end/src/components/navBar/navBar.scss

```diff
@@ -1,20 +1,30 @@
 @import '../../styles/colors.scss';
 
 .navBar {
-    flex-grow: 1;
+    display: flex;
+    align-items: center;
     background: $base-white;
+    height: 87px;
+    box-shadow: 0 3px 3px $shadow;
 }
 
-.icon {
-    width: "30px";
-    height: "30px";
+.logo {
+    padding-left: 30px;
+}
+
+.iconButton {
+    background-color: $base-white;
+    width: 45px;
+    height: 45px;
+    margin-right: 15px;
+    border: 0px;
```

Comment:
you don't need to add `px` if the value is `0`

Reply:
This is to overwrite native styling, which does add a border if you don't reset it to zero. Tested on both Safari and Chrome, I get borders around the button if you remove this line.